### PR TITLE
update the roadmap, add explanation of yml files

### PR DESCRIPTION
**Pull request by tangzhankun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 Ceph on Apache Mesos
 ======================
 A Mesos framework utilizing Docker for scaling a Ceph cluster. It aims to be a fast and reliable solution.  
-For now, it can only: 
-  - Start 1 mon, 3 osd, 1 radosgw. Each on a slave. Based on [ceph-docker]
-  - Accept Restful flexup request for launching a new osd to a new slave
+For now, it can only deploy ceph cluster on phisical machine:
```
Comment (zyluo) [inline README.md]: physical

File: README.md [hunk 2]
```diff
@@ -46,12 +46,40 @@ After that, you'll see "ceph-mesos", "ceph-mesos-executor", "ceph-mesos-tests" ,
 
 Run Ceph-Mesos
 --------------------------
-Modify the cephmesos.yml, you must populate the master and zookeeper fields, can leave other field default:
+Configure the cephmesos.yml and cephmesos.d/{hostname}.yml before you go.
+
+Ceph-Mesos chooses Macvlan to assign different IPs( DHCP needed) for OSDs in same host to work around the issue 19 mentioned in ceph-docker.And it supports disk management now. It can partition, mkfs.xfs and mount the disks for you.
```
Comment (zyluo) [inline README.md]: Maybe DHCP should be mentioned in the prerequisites section

Comment (zyluo) [inline README.md]: Do we still require 5 mesos slaves as described in the prerequisites section?

Comment (tangzhankun) [inline README.md]: no. No slaves count requirements. At least 1 I think

**Comment by tangzhankun** [comment]
I have updated, please check

File: README.md [hunk 3]
```diff
@@ -17,9 +17,10 @@ Scaling and monitoring large Ceph cluster in production Mesos environment in an
 
 Prerequisites
 --------------------------
-1. A Mesos cluster with Docker installed (duh). We only support CentOS 7 distribution at present and requires at least <b><em>5</em></b> slaves.
-2. Slaves in Mesos have network connection to download Docker images
-3. install libmicrohttpd in all slaves.
+1. A Mesos cluster with Docker installed (duh). We only support CentOS 7 distribution at present and requires at least <b><em>1</em></b> slave.
+2. DHCP server configured as Ceph-Mesos use Macvlan to assign IP to containers
```
Comment (zyluo) [inline README.md]: lets change to "DHCP service configured since ceph-Mesos use Macvlan to assign IP to containers."

File: README.md [hunk 4]
```diff
@@ -17,9 +17,10 @@ Scaling and monitoring large Ceph cluster in production Mesos environment in an
...
+3. Slaves in Mesos have network connection to download Docker images
+4. install libmicrohttpd in all slaves.
```
Comment (zyluo) [inline README.md]: "I"nstall

File: README.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 Ceph on Apache Mesos
 ======================
 A Mesos framework utilizing Docker for scaling a Ceph cluster. It aims to be a fast and reliable solution.  
-For now, it can only: 
-  - Start 1 mon, 3 osd, 1 radosgw. Each on a slave. Based on [ceph-docker]
-  - Accept Restful flexup request for launching a new osd to a new slave
+For now, it can only deploy ceph cluster on physical machine:
```
Comment (zyluo) [inline README.md]: let's not change this line. just makes it confusing. I don't think anyone will try ceph-mesos on vms

File: README.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
+  - Start 1 mon, 3 osd, 1 radosgw. Based on [ceph-docker]
```
Comment (zyluo) [inline README.md]: could we put a link on ceph-docker to the git repo

File: README.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
+  - Accept Restful flexup request for launching a new osd
```
Comment (zyluo) [inline README.md]: RESTful

**Comment by zyluo** [comment]
fix some nits and I think it's ready to merge

File: README.md (hunk 5 above)
Comment (tangzhankun) [inline README.md]: ok.

File: README.md (hunk 6 above)
Comment (tangzhankun) [inline README.md]: there already have link with it

File: README.md (hunk 3 above)
Comment (tangzhankun) [inline README.md]: ok

File: README.md (hunk 6 above)
Comment (zyluo) [inline README.md]: got it

**Comment by tangzhankun** [comment]
updated , check it please

**Pull request merged**